### PR TITLE
feat(config-ui): add jenkins no scope tips

### LINKED ISSUE #3457: [Feature]Config UI] Need to ask users to edit their old blueprints to add data scope for Jenkins
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-devlake/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

In the current Config UI, users don't need to select data scope for Jenkins and we collect all jobs. This logic is different from other plugins. So we want to ask users to select data scope for Jenkins, just like for other data sources.

The changes we are going to make:
1. When an old blueprint is about to run, we ask users to edit the blueprint by adding data scope for Jenkins, before the blueprint can run.
2. If a user is editing the blueprint, and no data scope is selected for Jenkins, he/she will not be able to save the blueprint.

### Use case

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by mintsweet**
### Summary
add jenkins no scope tips.

### Does this close any open issues?
Closes #3457

### CODE REVIEW

File: config-ui/src/pages/blueprint/detail/blueprint-detail-page.tsx [hunk 1]
```diff
@@ -41,6 +52,14 @@ export const BlueprintDetailPage = () => {
       ]}
     >
       <BlueprintDetail id={blueprint.id} />
+      {showJenkinsTips && (
+        <S.JenkinsTips>
+          <p>
+            Please add data scope for Jenkins before this Blueprint can run
+            again.
```
Comment (likyh) [inline config-ui/src/pages/blueprint/detail/blueprint-detail-page.tsx:59]: @Startrekzky look at this wording

**Review by likyh — DISMISSED** [review]

File: config-ui/src/pages/blueprint/detail/blueprint-detail-page.tsx (hunk 1 above)
Comment (Startrekzky) [reply config-ui/src/pages/blueprint/detail/blueprint-detail-page.tsx:59]: Please add the "Jenkins jobs" to collect before this Blueprint can run again.

**Review by Startrekzky — CHANGES_REQUESTED** [review]
Please change the wording of the tip.

**Review by likyh — APPROVED** [review]

**Review by Startrekzky — APPROVED** [review]
LGTM

**Pull request merged**